tracee-rules/engine/engine.go

```diff
@@ -62,24 +67,34 @@ func NewEngine(sigs []types.Signature, sources EventSources, output chan types.F
 	return &engine
 }
 
+// signatureStartFunc is the signature handling business logics.
+func (engine *Engine) signatureStartFunc(signature types.Signature, c chan types.Event) {
```

Comment:
does this func need to be attached to engine?

Reply:
nope, forgot to remove it.